### DISCUSSION TITLE
Stateless/Apolide/Apatride: not everyone has a citizenship!

### DISCUSSION THREAD

**Discussion by jfly** [thewca/worldcubeassociation.org#10505] [General]
So, this is actually a thing: https://en.wikipedia.org/wiki/Statelessness. The WCA Software Team just received an email from someone who identifies as stateless, so I've created this issue to discuss how to handle such people. Some questions I can think of:

1. How do we verify that someone is stateless?
2. How do we want to store statelessness? I don't believe NULL country is the right direction to go... we should probably create our own new Stateless "country".
3. What pages will need to change as a result of this? It wouldn't make sense to talk about national records for this country, for instance. It also wouldn't make sense to look at results for the Stateless nation...

(link to internal discussion with the Board: https://groups.google.com/d/msg/wca-admin/2BjWLYhMcfE/R0_oYgzPBQAJ)

**Comment by SAuroux** [comment]
Hey Jeremy,

I basically see two approaches here:
1. Neglect stateless people and ask them to register with the country they
   are living in as citizenship.
2. Add a "Stateless" country. In this case, such a person should now have
   and national/continental rankings, only world ranking on their profiles. In
   particular, there should not be national records for "Stateless".

2016-11-05 7:43 GMT+01:00 Jeremy Fleischman notifications@github.com:

**Comment by larspetrus** [comment]
On Fri, Nov 4, 2016 at 11:43 PM, Jeremy Fleischman <notifications@github.com

> wrote:
> - How do we verify that someone is stateless?

The general problem seems unsolvable. That's true for establishing regular
citizenship too, but far less so.

His email sounded like he is renouncing his existing citizenship. If so, a
copy of whatever paperwork makes that happen would be reasonable proof.

In order to not encourage people to claim statelessness for fun/attention,
I agree that we shouldn't keep track of records etc.

## 

Procrastinators - the Leaders of Tomorrow

**Comment by jfly** [comment]
Note, the WRC added [regulation 2e3](https://www.worldcubeassociation.org/regulations/#2e3):

> 2e3) Competitors who have no citizenship may compete as "Stateless".

I still think we should wait to actually add support for this until it is needed.
